CodeStarterCamp_Week3/Step1.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Step1.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by 홍성준 on 2023/08/02.
+//
+
+import Foundation
+
+class Person {
+    var name: String
+    var asset: Int = 0
+    init(name: String, asset: Int) {
+        self.name = name
+        self.asset = asset
+    }
+    
+    func buyCoffee(coffeeShop: CoffeeShop, menus: [Coffee]) {
+        coffeeShop.getOrder(menus: menus)
+        for menu in menus {
+            if let price = coffeeShop.menus[menu.rawValue] {
+                self.asset -= price
+            }
+        }
```

Comment:
여러개의 메뉴를 주문할 경우를 고려해서 작성해주셨네요 👍
전달인자 레이블을 활용하면 메서드를 호출할 때 좀 더 이해하기 쉬울 것 같습니다.
menu를 보통 복수형으로 쓰던가요...?! 💭

```suggestion
    func buyCoffee(menu: [Coffee], at coffeeShop: CoffeeShop) {
        coffeeShop.getOrder(menu: menu)
        for coffee in menu {
            if let price = coffeeShop.menu[coffee.rawValue] {
                self.asset -= price
            }
        }
```